### PR TITLE
Accuracy Debug

### PR DESCRIPTION
**Pull request by hnyls2002**
For debugging

With this shape

```py
    batch_size = 5
    prefix_lens = torch.tensor([128, 156, 127, 176, 243], dtype=torch.int32, device="cuda")
    seq_lens = torch.tensor([370, 390, 410, 234, 279], dtype=torch.int32, device="cuda")
```

The diff is
```
==================== Diff With Triton ====================
Mean:  tensor(0.0004, device='cuda:0', dtype=torch.bfloat16)
Max:  tensor(0.0117, device='cuda:0', dtype=torch.bfloat16)
==================== Diff With Ragged ====================
Mean:  tensor(0.0001, device='cuda:0', dtype=torch.bfloat16)
Max:  tensor(0.0078, device='cuda:0', dtype=torch.bfloat16)
```

With this shape

```py
    batch_size = 2
    prefix_lens = torch.tensor([261, 0], dtype=torch.int32, device="cuda")
    seq_lens = torch.tensor([367, 22], dtype=torch.int32, device="cuda")
```

The diff is

```
==================== Diff With Triton ====================
Mean:  tensor(0.0004, device='cuda:0', dtype=torch.bfloat16)
Max:  tensor(0.0156, device='cuda:0', dtype=torch.bfloat16)
==================== Diff With Ragged ====================
Mean:  tensor(0.0063, device='cuda:0', dtype=torch.bfloat16)
Max:  tensor(1.3203, device='cuda:0', dtype=torch.bfloat16)
```

### CODE REVIEW

**Comment by hnyls2002** [comment]
This issue has been solved by https://github.com/flashinfer-ai/flashinfer/pull/440, thanks to @yzh119
